build.sbt

```
@@ -1,26 +1,33 @@
+/*
+ * Copyright (c) 2020 Snowplow Analytics Ltd. All rights reserved.
```

```suggestion
 * Copyright (c) 2020-2020 Snowplow Analytics Ltd. All rights reserved.
```
Using yyyy-yyyy everywhere makes it easy to upgrade the licences via scripts

---

ok